cpop_bot/music_grabber.py

```diff
@@ -39,27 +39,24 @@ def download(self, input_url, filesize_limit):
 
         audio_filesize = self.__get_filesize(info_format)
         if int(audio_filesize) < filesize_limit:
-            logger.info(webpage_info + ": downloading...")
+            logger.info("%s: downloading...", webpage_info)
             ydl.process_info(info)
             info = self.__add_downloads_to_info_dict(info)
             if self.__files_successfully_downloaded(info['downloads']):
                 return info
-            else:
-                self.__delete_downloaded_files(info['downloads'])
-                raise ValueError(webpage_info +
-                                 ": failed to download audio or thumbnail")
-        else:
-            raise WrongFileSizeError(webpage_info + ": audio file size (" +
-                                     str(audio_filesize) + ") is greater" +
-                                     "than the limit: (" +
-                                     str(filesize_limit) + ")")
+            self.__delete_downloaded_files(info['downloads'])
+            raise ValueError(webpage_info +
+                             ": failed to download audio or thumbnail")
+        raise WrongFileSizeError(webpage_info + ": audio file size (" +
```

Comment:
it looks weird when you do it this way as if it's a default behivour. If you don't want to have **else** i think it's better to reverse the order and raise these errors right after the checks failed, so what i mean is do something like this:
```
if int(audio_filesize) > filesize_limit: 
          raise WrongFileSizeError(....)
....
# do your normal code of processing the audio
```

and then 
```
 if !self.__files_successfully_downloaded(info['downloads']):
         self.__delete_downloaded_files(info['downloads'])
         raise ValueError(webpage_info +
                             ": failed to download audio or thumbnail")
 return info
```

Reply:
force pushed the new commit which solved this.

Reply:
oh btw `!_self...` does not works, need to use `if not self...`